service/apiserver/server.api

```diff
@@ -34,10 +34,15 @@ type ReqGetRange {
 
 type (
 	AccountAsset {
-		Id       uint32 `json:"id"`
-		Name     string `json:"name"`
-		Balance  string `json:"balance"`
-		LpAmount string `json:"lp_amount"`
+		Id      uint32 `json:"id"`
+		Name    string `json:"name"`
+		Balance string `json:"balance"`
+		Price   string `json:"price"`
+	}
+
+	AccountLp {
+		Index  uint32 `json:"index"`
```

Comment:
would `PairIndex` be better?

Reply:
Because `AccountAsset` already uses `Id` (not `AssetId`), so i want to align `AccountAsset`.